### DISCUSSION TITLE
Making an existing category to be subcategory of another existing category

### DISCUSSION THREAD

**Discussion by Megidd** [saleor/saleor#7788] [Q&A] [answered]
I have three categories. 

`Parent`
`Child 0`
`Child 1`

# My situation

* Created some products with category of `Child 0` 
* Created some other products with category of `Child 1`
* By **mistake**, I forgot to create `Child 0` and `Child 1` as subcategory of `Parent`
* Organized the navigation bar in a way that  `Child 0` and `Child 1` are inside `Parent`

# Problem

The above setup causes a problem. When User clicks on the `Parent` on the navigation bar, no product is shown. Since, `Child 0` and `Child 1` are not actually subcategory of `Parent`.

I'm looking into `Product` category inside dashboard: there is no way to add an existing category as a subcategory.

# A possible solution

Of course, one solution is to create new subcategories like `Child 0 - new` and `Child 1 - new` as subcategories of `Parent`. Then modify all products by changing their categories from `Child 0` to `Child 0 - new` and from `Child 1` to `Child 1 - new`. Also modifying the navigation bar accordingly.

# Question

How can I fix my situation by an easier approach? Am I missing something?

**Comment by karolkielecki** [comment, marked as answer]
@Megidd Hi there. True--there is no way to convert an existing category to a subcategory of another category. I recommend you to try your provided solution.

Cheers,

**Reply by Megidd** [reply]
I did so and it fixed the problem :) Thanks!
